src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -460,19 +457,11 @@ void submitStreamCreateOperation(final PathOperation streamOp, final String node
 			val endpointUri = endpointCache.computeIfAbsent(nodeAddr, this::createEndpointUri);
 			val controller = controllerCache.computeIfAbsent(endpointUri, this::createController);
 			val scopeName = DEFAULT_SCOPE;
+			val scopeCreateFunc = scopeCreateFuncCache.computeIfAbsent(controller, ScopeCreateFunctionImpl::new);
+			val streamCreateFunc = streamCreateFuncCache.computeIfAbsent(scopeName, scopeCreateFunc);
+			val streamConfig = scopeStreamConfigsCache.computeIfAbsent(scopeName,streamCreateFunc);
 			var streamName = streamOp.item().name();
```

Comment:
let's use `val` and pass the `streamOp.item().name()` directly to the function fixing the stream name, without additional mutable local variable

Reply:
Ok. It must be better.